# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/moz.build
# Commit: e8bf96e8321a
# Full Hash: e8bf96e8321a7966be57bf567ce5aad7688f1e2e
# Author: Aaron Klotz <aklotz@mozilla.com>
# Date: 2019-07-24 09:54:28
# Regressor Bug: 1567614
# File Overlap Count: 1
# Description:
#   Bug 1567614: Part 1 - Refactor launcher process's LaunchUnelevated to delegate to ShellExecuteByExplorer; r=jmathies
#   
#   This is just so that both the launcher process and other Gecko code can share
#   this method.
#   
# ==============================================================================

diff -r 20e1a3ecd04a -r e8bf96e8321a widget/windows/moz.build
--- a/widget/windows/moz.build	Mon Jul 22 17:22:15 2019 +0000
+++ b/widget/windows/moz.build	Tue Jul 23 20:17:58 2019 +0000
@@ -19,6 +19,7 @@
 ]
 
 EXPORTS.mozilla += [
+    'ShellHeaderOnlyUtils.h',
     'WindowsConsole.h',
     'WinHeaderOnlyUtils.h',
 ]
